### ISSUE TITLE
Dynamic options

### ISSUE THREAD

**Issue by ilyakrasnov** [zefoy/ngx-perfect-scrollbar#45]
First of all, thanks for a great and so useful library @sconix 

I am trying to add settings dynamically - setting supressScrollY based on a constant. It looks like this: 

```typescript
<perfect-scrollbar  class="flex-item-elastic container" [config]="{suppressScrollY: !isWidget}">
    <router-outlet></router-outlet>
</perfect-scrollbar>
```

Though the constant changes, the config doesn't seem to take effect. I've also tried using a method call (`suppressScrollY: disableYScroll()`) - also without any effect, the method gets called though.

Any ideas how to achieve the desired effect?

Thanks a lot

**Comment by sconix** [comment]
Weird it seems that changing that setting dynamically does not have any effect. Let me look into this bit more and I will update this as soon as I find the reason.

**Comment by ilyakrasnov** [comment]
Thanks a lot!

**Comment by sconix** [comment]
It seems that I haven't added change detection for config changes. I will add it now and release a new version.

**Comment by sconix** [comment]
I just released 4.2.0 with support for config change detection, let me know if you still have problems.

**Comment by ilyakrasnov** [comment]
Thanks a lot @sconix! Works perfectly. Have a nice weekend
